From 93da5b422a64cf412f80eb5cec8f9858ddcff98a Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 16 Mar 2021 10:05:45 +0200
Subject: [PATCH] [SW-34382] preboot: greco: decrease sram

Dialing down the sram memory size for preboot, per greco's team
request

Change-Id: I3046872792928e3fd839412793aa51b18a3dca3b
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/greco/common.h b/greco/common.h
index 89738f3..5d0fd80 100644
--- a/greco/common.h
+++ b/greco/common.h
@@ -11,7 +11,7 @@
 #include <bitops.h>
 
 #define PPBOOT_SIZE			0x5000		/* 20KB */
-#define PREBOOT_SIZE			0x20000		/* 128KB */
+#define PREBOOT_SIZE			0x10000		/* 64KB */
 
 #define SRAM_SIZE			0x8000000	/* 128MB */
 #define HL_SP_SRAM_SIZE			0x10000
